homeassistant/components/environment_canada/camera.py

```diff
@@ -17,42 +17,34 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-ATTR_STATION = "station"
-ATTR_LOCATION = "location"
-ATTR_UPDATED = "updated"
-
 CONF_ATTRIBUTION = "Data provided by Environment Canada"
-CONF_STATION = "station"
 CONF_LOOP = "loop"
 CONF_PRECIP_TYPE = "precip_type"
 
 MIN_TIME_BETWEEN_UPDATES = datetime.timedelta(minutes=10)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
-        vol.Optional(CONF_LOOP, default=True): cv.boolean,
-        vol.Optional(CONF_NAME): cv.string,
-        vol.Optional(CONF_STATION): cv.matches_regex(r"^C[A-Z]{4}$|^[A-Z]{3}$"),
         vol.Inclusive(CONF_LATITUDE, "latlon"): cv.latitude,
         vol.Inclusive(CONF_LONGITUDE, "latlon"): cv.longitude,
-        vol.Optional(CONF_PRECIP_TYPE): ["RAIN", "SNOW"],
+        vol.Optional(CONF_LOOP, default=True): cv.boolean,
+        vol.Optional(CONF_NAME): cv.string,
+        vol.Optional(CONF_PRECIP_TYPE): ["rain", "snow"],
     }
```

Comment:
We no longer allow changes to YAML based configurations. This integration needs to be converted to use a configuration flow.

For more information see ADR-007 & ADR-0010.

<https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md>

and

<https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md>

Reply:
Okay, I can try to take a look at switching everything over. Would you be able to point me to a PR for an integration that's already been through this change? Looking through the documentation I'm a little bit lost.

Reply:
If you go to the Home Assistant frontend -> Configuration -> Integrations -> Hit the `+`. Every integration shown on that list has is implemented.